#Goal definition
float32[3] ini_point	#PreGrasp or PreUngrasp point
float32[3] grasp_point	#Grasp or ungrasp point
float32[3] end_point	#PostGrasp or PostUngrasp point
float32[3] ini_EF_rpy	#EF init Orientation
float32[3] end_EF_rpy	#EF end Orientation
bool pick				#Pick=1 or Place=0 action
bool execute			#Execute=false just calc IK. Execute=True to move robot
---
#Result definition
bool successful
---
#Feedback
int32 PorP_state	#State of the action

